### PR TITLE
docs: update kepler operator installation steps

### PR DESCRIPTION
**Pull request by vprashar2929**
This PR:
* Updates the installation/configuration steps for Kepler Operator on Kind
* Updates the installation steps for Kepler Operator using Community catalog(specific to `dev-preview`) on OpenShift.
* Updates the steps related to how to configure dashboards on OpenShift.

### CODE REVIEW

**Comment by husky-parul** [comment]
Thanks @vprashar2929 for updating this. I need to replicate this on local cluster before adding my reviews.

File: docs/installation/community-operator.md [hunk 1]
```diff
@@ -19,19 +19,21 @@ cd kepler-operator
 ## Remove previously installed version of the Kepler Community Operator
 
 If you have previously installed the Kepler Community Operator this will need
-to be removed prior to the installation of the `v0.6.z` version of the operator.
+to be removed prior to the installation of the `v0.8.z` version of the operator.
 This is due to updates to the Kepler API that not being backward compatible.
```
Comment (sthaha) [inline docs/installation/community-operator.md:23]:
```suggestion
This is due to changes to the Kepler API that are backward incompatible. 
Please also note that `v1alpha` does not promise backward compatibility and 
and backward incompatible changes are expected until the API matures to `v1beta1`.

```

File: docs/installation/community-operator.md [hunk 2]
```diff
@@ -45,50 +47,74 @@ kepler-operator.v0.5.0-230815081646   Kepler    0.5.0-230815081646
 
 ```
 
-2. To remove the exiting Kepler Community Operator use the following command,
-	 update the -v flag with the operator version that is installed.
+- To remove the exiting Kepler Community Operator use the following command,
```
Comment (sthaha) [inline docs/installation/community-operator.md:50]:
```suggestion
- To remove the existing Kepler Operator use the following command,
```

File: docs/installation/community-operator.md [hunk 3]
```diff
@@ -45,50 +47,74 @@ kepler-operator.v0.5.0-230815081646   Kepler    0.5.0-230815081646
...
+  update the -v flag with the operator version that is installed.
 
-```shell
-/hack/uninstall-operator.sh -v v0.5.0-230815081646 --delete
+```sh
+./hack/uninstall-operator.sh -v v0.5.0-230815081646 --delete
```
Comment (sthaha) [inline docs/installation/community-operator.md:54]:
```suggestion
./hack/uninstall-operator.sh -v v0.5.0 --delete
```

File: docs/installation/community-operator.md [hunk 4]
```diff
@@ -45,50 +47,74 @@ kepler-operator.v0.5.0-230815081646   Kepler    0.5.0-230815081646
...
-1. Go to Operators ❯ OperatorHub. Search for `Kepler`.
-   Click on Kepler Operator tile, then select `Continue` and then `Install`
+- Go to Operators ❯ OperatorHub. Search for `Kepler`.
+  Click on Kepler Operator tile, then select `Continue` and then `Install`
 
-![](../fig/ocp_installation/operator_installation_ocp_1_0.6.z.png)
+![](../fig/ocp_installation/operator_installation_ocp_1_0.8.z.png)
 
-2. No changes need to be made to the operator install options, select `Install`
+- Choose `dev-preview` channel for deploying `dev-preview` version of Operator and click on `Install`
```
Comment (sthaha) [inline docs/installation/community-operator.md:66]:
```suggestion
- Choose `alpha` or `dev-preview` channel for deploying the `latest` or the `developer preview` 
version of the Operator respectively.  
-  Click on `Install`
**NOTE:** `dev-preview` channel will only be updated until developer-preview is finalized.
```

File: docs/installation/community-operator.md [hunk 5]
```diff
@@ -45,50 +47,74 @@ kepler-operator.v0.5.0-230815081646   Kepler    0.5.0-230815081646
...
 
-![](../fig/ocp_installation/operator_installation_ocp_2_0.6.z.png)
+![](../fig/ocp_installation/operator_installation_ocp_2_0.8.z.png)
 
-2. Allow the operator to install
+- Wait till Operator gets installed
```
Comment (sthaha) [inline docs/installation/community-operator.md:70]:
```suggestion
- Wait until Operator gets installed
```

File: docs/installation/community-operator.md [hunk 6]
```diff
@@ -45,50 +47,74 @@ kepler-operator.v0.5.0-230815081646   Kepler    0.5.0-230815081646
...
+
+* Check that the Availability status of Kepler Instance should be `True`
+
+![](../fig/ocp_installation/operator_installation_ocp_6_0.8.z.png)
+
+- Check that the Kepler Exporter pods are running inside `openshift-kepler-operator` namespace
+
+```sh
+oc get pods --namespace openshift-kepler-operator
+```
```
Comment (sthaha) [inline docs/installation/community-operator.md:115]: Rather the user should be prompter to look at `Available` condition. 
`openshift-kepler-operator` namespace is an implementation detail and should in `Developer / Design / FAQ` docs than user guide.

Comment (sthaha) [reply docs/installation/community-operator.md:115]: `oc get kepler kepler -w` is the recommended way.

File: docs/installation/community-operator.md [hunk 7]
```diff
@@ -156,9 +198,8 @@ Refer to the [Grafana Deployment Overview](https://github.com/sustainable-comput
 
 ## Frequently Asked Questions
 
-# Will Kepler work on earlier releases of OpenShift
+### Will Kepler work on earlier releases of OpenShift?
```
Comment (sthaha) [inline docs/installation/community-operator.md:201]: Could we also add a section on 

* how do I set nodeSelectors and tolerations ?
* Where is kepler pod deployed?

File: docs/installation/kepler-operator.md [hunk 8]
```diff
@@ -1,21 +1,22 @@
...
+- Sign in as `kubeadmin` or a user with `cluster-admin` role
 
-### Run a kind cluster locally
+## Run a kind cluster locally
 
-``` sh
+```sh
 cd kepler-operator
-make cluster-up CLUSTER_PROVIDER='kind' CI_DEPLOY=true GRAFANA_ENABLE=true
+make cluster-up
```
Comment (sthaha) [inline docs/installation/kepler-operator.md:19]: Just `make fresh` should do

Comment (sthaha) [reply docs/installation/kepler-operator.md:19]: And we shoudn't require `kc get pods -n monitoring`

File: docs/installation/kepler-operator.md [hunk 9]
```diff
@@ -25,26 +26,34 @@ prometheus-operator-6bd88c8bdf-9f69h   2/2     Running   0          48m
 ```
 
 ## Run kepler-operator
+
 - You can use the image from quay.io to deploy kepler-operator.
 
 ```sh
-make deploy IMG=quay.io/sustainable_computing_io/kepler-operator:latest
-kubectl config set-context --current --namespace=monitoring
+make deploy OPERATOR_IMG=quay.io/sustainable_computing_io/kepler-operator:latest
```
Comment (sthaha) [inline docs/installation/kepler-operator.md:33]: `make fresh` already builds the operator, pushes it to local repo. Devs just need to run the `operator-sdk run bundle` command.

**Review by sthaha — CHANGES_REQUESTED** [review]

**Comment by sthaha** [comment]
@vprashar2929 Thanks a lot for the docs updates, quite helpful. Some comments to consider.

File: docs/installation/kepler-operator.md (hunk 8 above)
Comment (husky-parul) [reply docs/installation/kepler-operator.md:19]: When I run `make cluster-up`  I see the following
```
 🔆🔆🔆  Ensuring all tools are installed  🔆🔆🔆 
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 🔔 installing all tools ...
 🔔 installing controller-gen with version: v0.12.1
go: downloading sigs.k8s.io/controller-tools v0.12.1
go: downloading github.com/spf13/cobra v1.7.0
go: downloading k8s.io/api v0.27.1
go: downloading k8s.io/apimachinery v0.27.1
```
But the command failed on a machine with no pre-installed  `kind` 

```
Starting kind with cluster name "kind"

 ❯ kind create cluster --name=kind -v6 --config=/home/parsingh/go/src/github.com/test-pr/kepler-operator/tmp/local-dev-cluster/tmp/kind/kind.yml

/home/parsingh/go/src/github.com/test-pr/kepler-operator/tmp/local-dev-cluster/lib/utils.sh: line 50: kind: command not found
make: *** [Makefile:120: cluster-up] Error 127
```

Shouldn't the `make` command ensures that we have kind pre-installed as well?
I will open a corresponding issue in the `kepler-operator`

File: docs/installation/community-operator.md [hunk 10]
```diff
@@ -19,134 +19,180 @@ cd kepler-operator
...
+![](../fig/ocp_installation/operator_installation_ocp_6_0.8.z.png)
 
-5. Check that the Kepler Exporter pods are running in the `openshift-kepler-operator` namespace
+- Check that the Kepler is deployed and available
 
-```shell
-oc project openshift-kepler-operator
-oc get pods
+```sh
+oc get kepler kepler
```
Comment (sthaha) [inline docs/installation/community-operator.md:119]:
```suggestion
oc get kepler kepler -w
```

**Review by sthaha — APPROVED** [review]

**Pull request merged**